include/rocksdb/options.h

```diff
@@ -1600,6 +1600,21 @@ struct ReadOptions {
   // Default: false
   bool adaptive_readahead;
 
+  // For file reads associated with this operation, charge the internal rate
+  // limiter (see `DBOptions::rate_limiter`) at the specified priority. The
+  // special value `Env::IO_TOTAL` disables charging the rate limiter.
+  //
+  // The rate limiting is bypassed no matter this option's value for
+  // plain/cuckoo table reads, old BlobDB reads, and generally small reads like
+  // for file headers/footers.
+  //
+  // APIs that rely on `RandomAccessFile::MultiRead()`, such as
+  // `DB::MultiGet()`, return `Status::NotSupported` when
+  // `priority != Env::IO_TOTAL`.
```

Comment:
I am thinking maybe a more user-relevant way of stating this can be:

```
List of lower-level reads that support rate-limiting and take ReaderOptions::priority into account include: X, Y, Z (list is subjected to future expansion). 
For others, there isn't any rate-limiting and ReaderOptions::priority will either be ignored (e.g, plain/cuckoo table reads, old BlobDB reads, and generally small reads like or file headers/footers) or requires ReaderOptions::priority to be Env::IO_TOTAL (e.g, RandomAccessFile::MultiRead()/DB::MultiGet()) .
```